actions/tag-and-release/action.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Tag and release"
+description: "Tag and release your repo with target tag."
+
+inputs:
+  tag:
+    description: "The version of the tag to be publish and released e.g 1.0.0"
+    required: true
+  release-title:
+    description: "Text for the github release title"
+    required: true
+  release-body:
+    description: "Text for the github release body (this describes the release content / Changelog)"
+    default: ""
+    required: false
+  github-username:
+    description: "The GitHub username for committing the changes."
+    required: true
+  github-email:
+    description: "The GitHub email for committing the changes."
+    required: true
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: Create tag
+      run: |
+        git config --local user.name ${{ inputs.github-username }}
+        git config --local user.email ${{ inputs.github-username }}
+        git tag v${{ inputs.tag }}
+      shell: bash
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v1
+      env:
+        GITHUB_TOKEN: ${{ inputs.github-token }}
+      with:
+        tag_name: "${{ inputs.tag }}"
```

Comment:
wouldn't the tag name also need the `v` prefix here?

Reply:
we don't want to name the tags https://github.com/bakdata/kpops/blob/main/.github/workflows/release.yaml#L50 like here

Reply:
but doesn't the previous step do `git tag v...`?

Reply:
not sure about it too xD